.github/workflows/build-push-docker-image.yml

```diff
@@ -0,0 +1,36 @@
+name: Publish Docker image
+
+on:
+  release:
+    types: [published]
```

Comment:
Shall we change it also in `publish package to npmjs`? Or maybe we should have one workflow `publish.yaml` with two jobs?

Reply:
Yea, I think it is a better option since they do the same, just to different repositories